tests/test_stack.py

```diff
@@ -41,3 +41,12 @@ def test_items_removed_in_lifo_order(stack):
     assert stack.pop() == 3
     assert stack.pop() == 5
     assert stack.empty()
+
+def test_str_with_items(stack):   
+    stack.push(5)
+    stack.push(3)
+    stack.push(7)
+    assert stack.__str__() == [7, 3, 5]
```

Comment:
Rather than calling `__str__` directly, usually we use the `str()` method, like `str(stack)`.

Also, notice that the value you're comparing to isn't a string, it's a list. But the `__str__` method should be returning a string. So a more accurate assert line might be

```suggestion
    assert str(stack) == "[7, 3, 5]"
```